### PR TITLE
Integrate algobpy into sample-project

### PR DESCRIPTION
**Pull request by ratik21**
[https://www.pivotaltracker.com/story/show/177791730](https://www.pivotaltracker.com/story/show/177791730)

### CODE REVIEW

File: packages/algob/sample-project/assets/escrow.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from pyteal import *
+
+def escrow_contract(RECEIVER_ADDRESS):
+    '''
+    This contract represents an escrow account which only approves transfer to
+    address "RECEIVER_ADDRESS"
+    '''
+
+    # verify neither transaction
+    # contains a rekey
```
[the remark this answers is not in the document]
Comment (ratik21) [reply packages/algob/sample-project/assets/escrow.py:16]: yes

File: packages/algob/sample-project/scripts/2-escrow-account.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * In this script we will fund an escrow contract. The escrow contract
+ * ensures the payment is made out(from the escrow) to a specific receiver only.
+ * This receiver address is hardcoded in the smart contract, and can be passed
+ * dynamically to the contract using fundLsig function (passed as a template parameter)
+ */
+async function run (runtimeEnv, deployer) {
+  console.log('Sample script for escrow account started execution!');
```
[the remark this answers is not in the document]
Comment (ratik21) [reply packages/algob/sample-project/scripts/2-escrow-account.js:8]: yes

File: packages/algob/sample-project/scripts/2-escrow-account.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  // RECEIVER_ADDRESS is set in escrow.py when it is compiled from PyTEAL to TEAL
+  const templateParams = {
+    RECEIVER_ADDRESS: 'WHVQXVVCQAD7WX3HHFKNVUL3MOANX3BYXXMEEJEJWOZNRXJNTN7LTNPSTY'
+  };
+  await deployer.fundLsig('escrow.py',
+    { funder: deployer.accounts[0], fundingMicroAlgo: 20e6 }, {}, templateParams);
+
+  await deployer.addCheckpointKV('User Checkpoint Escrow', 'Fund Escrow Account');
```
[the remark this answers is not in the document]
Comment (ratik21) [reply packages/algob/sample-project/scripts/2-escrow-account.js:17]: added

**Review by amityadav0 — APPROVED** [review]

**Comment by amityadav0** [comment]
Please rename `parseTemp` to `parse_params`

File: packages/algob/sample-project/algobpy/parse.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import sys
+import yaml
+
+'''
+Overwrites scParam values if args is defined and has keys common from scParam
+'''
+def parseArgs(args, scParam):
```
Comment (amityadav0) [inline packages/algob/sample-project/algobpy/parse.py:7]: Please rename this function to `parse_params`

Comment (ratik21) [reply packages/algob/sample-project/algobpy/parse.py:7]: done

**Pull request merged**